src/pages/proposals/fundings.js

```diff
@@ -57,10 +57,21 @@ class ProposalFundings extends React.Component {
     const milestoneFundDifference = this.getMilestonFundDifference(milestoneFunds);
     const rewardDifference = this.getRewardDifference();
 
-    let reward = finalReward.original;
-    reward = truncateNumber(reward);
     milestoneFunds = truncateNumber(milestoneFunds);
 
+    let reward;
+    if (typeof finalReward === 'undefined') {
```

Comment:
`typeof x === 'undefined'` is a bit old fashioned. 

If you need to test for truthiness or existence, you can simply do `if (finalReward) {`.

Now if `finalReward` can be `0` this condition can be augmented with `if (finalReward || finalReward === 0)`. Either  way, that assumption should be checked.

Reply:
@FrancisMurillo yeah that was actually my initial thought, but decided to use `typeof` because i'm just checking for `undefined`, and don't wanna catch `null`, `0` or any other non-existent or non-truth type, but yeah will change it, thanks :+1: